crates/holochain/tests/sharded_gossip/mod.rs

```diff
@@ -307,34 +312,50 @@ async fn test_zero_arc_no_gossip_4way() {
 
     let configs = [
         // Standard config
-        TestConfig {
+        <TestConfig as Into<SweetConductorConfig>>::into(TestConfig {
             publish: true,
             recent: true,
             historical: true,
             bootstrap: true,
             recent_threshold: None,
-        }
-        .into(),
+        })
+        .tune_conductor(|params| {
+            // Speed up sys validation retry when gets hit a conductor that isn't yet serving the requested data
+            params.sys_validation_retry_delay = Some(std::time::Duration::from_millis(100));
+        }),
         // Publishing turned off
-        TestConfig {
+        <TestConfig as Into<SweetConductorConfig>>::into(TestConfig {
```

Comment:
:nauseated_face:

Reply:
Yes, it's not the prettiest change here, just took what the compiler recommended. I slightly prefer this over modifying the `TestConfig` struct to include more fields because it's quite focused at the moment